builtin/bigint_nonjs.mbt

```diff
@@ -876,3 +876,41 @@ pub fn BigInt::to_octets(self : BigInt, length? : Int) -> Bytes {
   }
   result
 }
+
+///|
+pub fn land(self : BigInt, other : BigInt) -> BigInt {
+  let new_limbs = FixedArray::make(self.limbs.length(), 0U)
+  for i = 0; i < self.len; i = i + 1 {
+    new_limbs[i] = self.limbs[i] & other.limbs[i]
```

Comment:
The implementation does not seem to take into account the length of the two inputs.
With
```
  // large input
  let a = BigInt::from_string("123456789012345678901234567890")
  inspect!(a & 0N, content="0")
```
We have: `RuntimeError: array element access out of bounds`

Reply:
Pushed the new implementation that makes two BigInt's limbs when calculating same size long. Should fix this.